File: ProjectEuler/p7.java
public class p7 {
public static void main(String[]args) {
	int count = 0;

		for(int i = 2; i <= 5000000; i++) {
			if(isPrime(i)) count++;
			if(count == 10001) {
				System.out.println(i);
				break;
				}
			}

    }

    public static boolean isPrime(int x) {
    	for(int i = 2; i <= Math.sqrt(x); i++) {
    		if(x%i == 0) return false;
    	}
    	return true;
    }

}

File: ProjectEuler/Python ProjectEuler/p7.py
def isPrime(x):
  for i in range(2, int(x**.5)+1):
    if x % i == 0: return False
  return True

i,x = 2,1

while x != 10001:
  i += 1
  if isPrime(i): x += 1

print(i)